taca/server_status/cli.py

```diff
@@ -37,8 +37,11 @@ def uppmax(disk_quota, cpu_hours):
 		merged_results.update(disk_quota_data)
 	if cpu_hours:
 		cpu_hours_data = status.get_uppmax_cpu_hours()
-		for key in cpu_hours_data.keys():
-			merged_results[key].update(cpu_hours_data[key])
+		if not merged_results:
```

Comment:
I'd prefer to avoid using `try - except` for such checks. But you're right, it's better to put this check inside of the loop